public/whitepaper.htm

```diff
@@ -1,6 +1,8 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+    <title>Airswap Whitepaper - Peer to peer decentralized exchange</title>
+    <meta name="description" content="Learn about the peer to peer decentralized swap protocol from Airswap.">
```

Comment:
I think it should be identical to the others: "AirSwap: Peer-to-peer Token Trading DEX and Open Source Developer DAO"